CShartOOP/Homework1.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System.Globalization;
+
+namespace Kristina_Kulich__Application.CShartOOP;
+
+public class Homework1
+{
+        public void Calculatechecksum(int amount, double price, double sale)
+        {
+            double sum = amount * price - (sale*0.01);
+            double sumDiscounted = sum - sum * 0.01;
+            Console.WriteLine($"The final sum is:{sumDiscounted}");
```

Comment:
Please review the task. Method should return final sum, not void + console output